src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee

```diff
@@ -108,6 +110,9 @@ DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations
           $scope.apps = MnoeAppInstances.appInstances
       )
 
+  $scope.displayDock = ->
```

Comment:
Add a comment explaining why you hide the dock?
`# Hide the dock when no apps and marketplace is disabled`